### PR TITLE
[ENH] MDS: Show Kruskal stress

### LINKED ISSUE #6070: MDS: amount of explained variance
**What's your use case?**

I got a comment from a reviewer, that it would be useful if MDS showed the amount of explained variable. Given that most MDS projections start with PCA initialization, this could perhaps be useful.

**What's your proposed solution?**

Add info on amount of explained variance. Perhaps to the status bar or to the info box below parameter settings.

**Are there any alternative solutions?**
Look for the same info with PCA widget?

### PR DESCRIPTION
**Pull request by janezd**
##### Issue

Fixes #6070.

##### Description of changes

To reviewer: please check that I indeed called `update_stress` from all places where it's needed -- and that it's not called multiple times.

I don't know where to show the stress. Optimization box was about the best place. I don't want to put it on the graph, because then we'd need to at least allow the user to move it and ... no.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/widgets/unsupervised/owmds.py [hunk 1]
```diff
@@ -200,6 +200,7 @@ def __init__(self):
 
         self.embedding = None  # type: Optional[np.ndarray]
         self.effective_matrix = None  # type: Optional[DistMatrix]
+        self.stress = None
```
Comment (lanzagar) [inline Orange/widgets/unsupervised/owmds.py:203]: As shown by failing tests, this conflicts with the static method with the same name.
This results in a crash when setting the size of points to "Stress".

**Review by lanzagar — CHANGES_REQUESTED** [review]

File: Orange/widgets/unsupervised/owmds.py (hunk 1 above)
Comment (janezd) [reply Orange/widgets/unsupervised/owmds.py:203]: It's not that I don't warn students about this exact mistake.

File: Orange/widgets/unsupervised/owmds.py [hunk 2]
```diff
@@ -392,17 +395,32 @@ def on_partial_result(self, result: Result):
...
+        stress_val = "-" if self.stress is None else f"{self.stress:.3f}"
+        self.stress_label.setText(f"Kruskal Stress: {stress_val}")
+
+    def _compute_stress(self):
+        if self.embedding is None or self.effective_matrix is None:
+            return None
+        actual = scipy.spatial.distance.pdist(self.embedding)
+        actual = scipy.spatial.distance.squareform(actual)
+        return np.sqrt(np.sum((actual - self.effective_matrix) ** 2)
+                       / (np.sum(self.effective_matrix ** 2) or 1))
```
Comment (lanzagar) [inline Orange/widgets/unsupervised/owmds.py:422]: It would probably be better to call `get_stress` and normalize the returned result.

Comment (janezd) [reply Orange/widgets/unsupervised/owmds.py]: Done.

**Pull request merged**